spec/racker_spec.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+RSpec.describe Racker do
+  let(:app) { Rack::Builder.parse_file('config.ru').first }
+  let(:game) { Codebreaker::Game.new }
+
+  context 'rules path' do
+    before { get '/rules' }
+
+    it { expect(last_response).to be_ok }
+    it { expect(last_response.body).to include I18n.t(:rules_test) }
+  end
+
+  context 'unknown path' do
+    before { get '/unknown' }
+
+    it { expect(last_response).to be_not_found }
+    it { expect(last_response.body).to include I18n.t(:not_found) }
+  end
+
+  context 'statistics path' do
+    before { get '/stat' }
+
+    it { expect(last_response).to be_ok }
+    it { expect(last_response.body).to include I18n.t(:codebreak_year) }
+    it { expect(last_response.body).to include I18n.t(:top_of_player) }
+  end
+
+  context 'game' do
+    it 'makes number' do
+      post '/game', number: '1234', level: 'easy', player_name: 'Test'
+      expect(last_request.params['player_name']).to be_kind_of(String)
+      expect(last_request.params['level']).to be_kind_of(String)
+      expect(last_request.params['number']).to be_kind_of(String)
+      expect(last_response.body).to include last_request.params['player_name']
+    end
+  end
+
+  context 'submit_answer' do
+    it 'sub answer' do
+      game.new_game
+      game.enter_level('easy')
+      game.enter_name('Test')
+      env 'rack.session', game: game
+      post '/submit_answer', number: game.secret_code.join
+      expect(last_response.body).to include I18n.t(:won, name: 'Test')
+    end
+  end
+
+  context 'root path' do
+    before { get '/' }
+
+    it { expect(last_response).to be_ok }
+    it { expect(last_response.body).to include I18n.t(:codebreak_year) }
+    it { expect(last_response.body).to include I18n.t(:guess_number) }
+  end
+
+  context 'win' do
+    it 'win' do
```

Comment:
Only game should redirect user for this page. But now
https://pacific-basin-49824.herokuapp.com/win
https://pacific-basin-49824.herokuapp.com/lose
Handle it with a redirect or something else